BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/ble/beacon/signal/MeanFilter.java

```diff
@@ -11,21 +11,20 @@
 
 public class MeanFilter extends WindowFilter {
 
-    public MeanFilter(long duration, TimeUnit timeUnit) {
-        super(duration,timeUnit);
+    public MeanFilter() {
```

Comment:
You will not be able to create an `MeanFilter` with custom `duration` and `timeUnit` on construction.